source/fundamentals/collations.txt

```diff
@@ -0,0 +1,280 @@
+called ``contacts`` in a database called ``myproject`` using a German collation
+with the ``locale`` parameter set to ``de``.
+
+.. literalinclude:: /code-snippets/collation/find-sort-collation.js
+   :language: javascript
+   :dedent: 4
+
+findOneAndUpdate() Example
+``````````````````````````
+
+The following example calls the ``findOneAndUpdate()`` operation on a
+collection called ``names`` in which contains the following documents:
+
+::
+
+  { "_id" : 1, "first_name" : "Hans" }
+  { "_id" : 2, "first_name" : "Gunter" }
+  { "_id" : 3, "first_name" : "Günter" }
+  { "_id" : 4, "first_name" : "Jürgen" }
+
+Consider the following ``findOneAndUpdate()`` operation on this collection
+which **does not** specify a collation:
+
+.. literalinclude:: /code-snippets/collation/findOneAndUpdate-collation.js
+   :language: javascript
+   :start-after: start findOneAndUpdate without collation
+   :end-before: end findOneAndUpdate without collation
+   :dedent: 4
+
+Since "Gunter" is the first result in the sorted results when using a binary
+collation, the operation does not update any documents. Consider the same
+operation with a collation specified with the locale set to
+``de@collation=phonebook``. This locale specifies the ``collation=phonebook``
+option which contains rules for prioritizing proper nouns. The
+``de@collation=phonebook`` locale and option sorts characters with umlauts
+before the same characters without umlauts.
+
+.. literalinclude:: /code-snippets/collation/findOneAndUpdate-collation.js
+   :language: javascript
+   :start-after: start findOneAndUpdate with collation
+   :end-before: end findOneAndUpdate with collation
+   :dedent: 4
+
+The ``findOneAndUpdate()`` operation above returns the following updated
+document:
+
+::
+
+  { lastErrorObject: { updatedExisting: true, n: 1 },
+    value: { _id: 3, first_name: 'Günter' },
+    ok: 1 }
+
+findOneAndDelete() Example
+``````````````````````````
+
+The following example calls the ``findOneAndDelete()`` operation on a
+collection named ``numbers`` which contains the following documents:
+
+::
+
+  { "_id" : 1, "a" : "16" }
+  { "_id" : 2, "a" : "84" }
+  { "_id" : 3, "a" : "179" }
+
+In this example, we set the ``numericOrdering`` collation parameter to ``true``
+to sort numeric strings based on their numerical order instead of their
+lexical order.
+
+.. literalinclude:: /code-snippets/collation/findOneAndDelete-collation.js
+   :language: javascript
+   :start-after: start findOneAndDelete example with collation
+   :end-before: end findOneAndDelete example with collation
+   :dedent: 4
+
+After you run the operation above, the collection contains the following
+documents:
+
+::
+
+  { "_id" : 1, "a" : "16" }
+  { "_id" : 2, "a" : "84" }
+
+If you perform the same operation without collation on the original
+collection of three documents, the server deletes the first document it
+finds in which the lexical value of "``a``" is greater than the lexical value
+of "``100``".
```

Comment:
Calling out the exact `a` value that is deleted might be helpful here and below.

Reply:
I guess I need to clarify this. It's not "a", but rather the value assigned to "a" which hopefully is apparent when listing each of the values of "a". 
Perhaps "the lexical value of the "a" field (e.g. ...)" ?

Reply:
Your suggested change is definitely in the right direction, but my point was moreso that pointing out the actual deleted value could help readers follow along better -- I felt doing the manual diff (and scrolling up quite a ways to check the contents of the original collection) was a bit of a chore as a reader.